### PR TITLE
Enable referencing processingStepsMeta from eregistrations

### PR DESCRIPTION
**Pull request by kamsi**
It would be great convenience if we didn't have to pass `processingStepsMeta` to functionalities within eregistrations.

### CODE REVIEW

**Comment by kamsi** [comment]
@medikoo let's traditionally approve this one before I go deeper into adjusts.

File: processing-steps-meta.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+// It is meant to be mutated by the end system. It's here to facilitate referencing it.
+
+module.exports = {};
```
Comment (medikoo) [inline processing-steps-meta.js]: No need for this (`module.exports` is by default an empty object)

File: server/routes/official.js [hunk 2]
```diff
@@ -336,12 +336,11 @@ module.exports = exports = function (mainConf/*, options */) {
 	recentlyVisitedContextName = options.recentlyVisitedContextName;
 
 	/**
-	 * This is to safeguard older systems which don't use this functionality
-	 * the flag is not mandatory, so only setup if you don't want this configured in a system
+	 * This is to safeguard older systems which don't use this functionality.
 	 * */
-	if (options.processingStepsMeta) {
+	if (!Object.keys(processingStepsMeta).length) {
```
Comment (medikoo) [inline server/routes/official.js]: I think it should be `if (Object.keys(processingStepsMeta).length)`

File: server/scripts/db-recompute/index.js [hunk 3]
```diff
@@ -2,19 +2,19 @@
...
+  , ensureIterable      = require('es5-ext/iterable/validate-object')
+  , ensureString        = require('es5-ext/object/validate-stringifiable-value')
+  , Map                 = require('es6-map')
+  , Set                 = require('es6-set')
+  , deferred            = require('deferred')
+  , debug               = require('debug-ext')('setup', 4)
+  , ensureDriver        = require('dbjs-persistence/ensure-driver')
+  , recompute           = require('dbjs-persistence/recompute')
+  , isOfficialRoleName  = require('../../../utils/is-official-role-name')
+  , copyIsReady         = require('../../services/copy-is-ready');
```
Comment (medikoo) [inline server/scripts/db-recompute/index.js]: This whitespace changes seems wrong

File: server/routes/official.js (hunk 2 above)
Comment (kamsi) [inline server/routes/official.js]: Oh yes... not really tested (would come out in context of lomas)

**Comment by kamsi** [comment]
@medikoo I tested lomas, nothing looks broken (I checked the paginated lists in part b).

**Comment by mtuchowski** [comment]
All adapts tested. No bugs detected.

**Pull request merged**